draft-terbu-sd-jwt-vc.md

```diff
@@ -175,72 +198,71 @@ The following is a non-normative example of a decoded SD-JWT header:
 
-```
-{
-  "type": "SomeType"
-}
-```
+#### Registered JWT Claims
 
-#### Usage of registered JWT Claims
+VC-SD-JWTs MAY use any claim registered in the "JSON Web Token Claims"
+registry as defined in [@!RFC7519].
 
-The following are non-selectively disclosable registered JWT claims that
-SD-JWTs of VC-SD-JWTs contain for specific purposes:
+If present, the following registered JWT claims MUST be included in the SD-JWT
+and MUST NOT be included in the Disclosures, i.e. cannot be selectively
+disclosed:
 
 * `iss`
-    * REQUIRED. The issuer of the Verifiable Credential. The value of `iss`
+    * REQUIRED. The Issuer of the Verifiable Credential. The value of `iss`
 MUST be a URI. See [@!RFC7519] for more information.
 * `iat`
-    * REQUIRED. The time of issuance of the Verifiable Credential. See [@!RFC7519]
-for more information.
+    * REQUIRED. The time of issuance of the Verifiable Credential. See
+[@!RFC7519] for more information.
 * `nbf`
-    * REQUIRED. The time before which the Verifiable Credential MUST NOT be
+    * OPTIONAL. The time before which the Verifiable Credential MUST NOT be
 accepted before validating. See [@!RFC7519] for more information.
 * `exp`
-    * REQUIRED. The expiry time of the Verifiable Credential after which the
-proof of the Verifiable Credential is no longer valid. See [@!RFC7519] for more
+    * OPTIONAL. The expiry time of the Verifiable Credential after which the
+Verifiable Credential is no longer valid. See [@!RFC7519] for more
 information.
 * `cnf`
-    * OPTIONAL. The confirmation method can be used to verify the holder
-binding of the Verifiable Presentation. See [@!RFC7800] for more information.
+    * OPTIONAL. The confirmation method which can be used to verify the Holder
+Binding of the Verifiable Presentation. See [@!RFC7800] for more information.
 * `type`
     * REQUIRED. The type of the Verifiable Credential, e.g.,
-`IdentityCredential`. The claims belonging to the type can be contained in
-different components of the VC-SD-JWT, e.g., SD-JWT or Disclosures.
+`IdentityCredential`, as defined in {type-claim}.
 * `status`
     * OPTIONAL. The information on how to read the status of the Verifiable
-Credential. See TBD for more information.
+Credential. See [TBD] for more information.
 
-The following are selectively disclosable registered JWT claims that
-SD-JWTs of VC-SD-JWTs contain for specific purposes:
+The following registered JWT claims MAY be contained in the SD-JWT or in the
+Disclosures and MAY be selectively disclosed:
 
 * `sub`
     * OPTIONAL. The identifier of the Subject of the Verifiable Credential.
 The value of `sub` MUST be a URI. The Issuer MAY use it to provide the Subject
-identifier assigned and maintained by the Issuer. There is no requirement for
-a binding to exist between `sub` and `cnf` claims.
+identifier assigned by the Issuer. There is no requirement for a binding to
+exist between `sub` and `cnf` claims.
+
+#### `type` claim {#type-claim}
```

Comment:
type is now used before it is actually defined

Reply:
fixed